### ISSUE TITLE
2024 Primary Election Precinct Results

### ISSUE THREAD

**Issue by dwillis** [openelections/openelections-data-pa#153]
Using [Tabula](http://tabula.technology/), OCR or whatever method you can, parse precinct-level results for the following counties. Original sources are in the [sources-pa repository](https://github.com/openelections/openelections-sources-pa/tree/master/2024/primary).

The goal is to create a single CSV file for each county, with the following headers:

`county`, `precinct`, `office`, `district`, `party`, `candidate`, `votes`

If the county file also provides a breakdown of votes by method, include that using the following headers:

`early_voting`, `election_day`, `provisional`, `absentee`

Include the following offices:

* Registered Voters (if available)
* Ballots Cast (if available)
* President
* U.S. Senate
* U.S. House
* State Senate
* General Assembly

The CSV files should be named `20240423__pa__primary__{county}__precinct.csv`. Here's an example finished file: https://github.com/openelections/openelections-data-pa/blob/master/2022/20221108__pa__general__chester__precinct.csv.

- [ ] Adams
- [ ] Allegheny
- [ ] Armstrong
- [ ] Beaver
- [ ] Bedford
- [ ] Berks
- [ ] Blair
- [ ] Bradford
- [ ] Bucks
- [ ] Butler
- [ ] Cambria
- [ ] Cameron
- [ ] Carbon
- [ ] Centre
- [ ] Chester
- [ ] Clarion
- [ ] Clearfield
- [ ] Clinton
- [ ] Columbia
- [ ] Crawford
- [ ] Cumberland
- [ ] Dauphin
- [ ] Delaware
- [ ] Elk
- [ ] Erie
- [ ] Fayette
- [ ] Forest
- [ ] Franklin
- [ ] Fulton
- [ ] Greene
- [ ] Huntingdon
- [ ] Indiana
- [ ] Jefferson
- [ ] Juniata
- [ ] Lackawanna
- [ ] Lancaster
- [ ] Lawrence
- [ ] Lebanon
- [ ] Lehigh
- [ ] Luzerne
- [ ] Lycoming
- [ ] McKean
- [ ] Mercer
- [ ] Mifflin
- [ ] Monroe
- [ ] Montgomery
- [ ] Montour
- [ ] Northampton
- [ ] Northumberland
- [ ] Perry
- [ ] Philadelphia
- [ ] Pike
- [ ] Potter
- [ ] Schuylkill
- [ ] Snyder
- [ ] Somerset
- [ ] Sullivan
- [ ] Susquehanna
- [ ] Tioga
- [ ] Union
- [ ] Venango
- [ ] Warren
- [ ] Washington
- [ ] Wayne
- [ ] Westmoreland
- [ ] Wyoming
- [ ] York

**Comment by declanrjb** [comment]
If additional races are available (Attorney General, State Treasurer, etc), should I include them?

**Comment by dwillis** [comment]
> If additional races are available (Attorney General, State Treasurer, etc), should I include them?

Yes, please! Should have put that in the issue - all statewide races.
